ruleset/mutable.md

```diff
@@ -78,3 +80,15 @@ This rule takes precedence over every other rule determining the winner.
 ## 301
 
 Players may not subtract 291 from the ordinal number of their proposal and multiply the result by the fraction of favorable votes it received, rounded to the nearest integer. No rule with a lower ordinal number can take precedence over this rule.
+
+## 302
+
+Amend 202 to read:
+
+```
+One turn consists of two parts in this order: (1) proposing one rule-change and having it voted on, and (2) throwing one die once and adding the number of points on its face to one's score.
+
+In mail and computer games, instead of throwing a die, players subtract 202 from the ordinal number of their proposal and multiply the result by the fraction of favorable votes it received, rounded to the nearest integer. In the event that there is a dispute about which integer is nearest, https://github.com/ereddrex will decide which integer is nearest. 
+```
+
+202 will be updated directly in the github repo to indicate that it has been amended. 
```

Comment:
Seeing as the changes include the update to 202 to include the link (nice touch), I don't think we need this line stating that 202 will be updated, right?
```suggestion

```